spec/lib/guard/notifier_spec.rb

```diff
@@ -41,4 +44,27 @@
       end
     end
   end
+
+  describe ".notify" do
+
+    before do
+      subject.connect
+      allow(notifier).to receive(:notify)
+    end
+
+    context "with no options" do
+      it "notifies" do
+        expect(notifier).to receive(:notify)
```

Comment:
While the `allow()` is just to "stub" the notifier, `expect()` should actually check parameters (`.with("A")`).